packages/trader/src/Modules/Trading/Components/Form/TradeParams/Duration/duration-range-text.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Text } from '@deriv/components';
+import { addComma } from '@deriv/shared';
+import { Localize } from '@deriv/translations';
+
+type TDurationRangeText = {
+    min: string;
+    max: string;
+    duration_unit: string;
+};
+
+const DurationRangeText = ({ min, max, duration_unit }: TDurationRangeText) => {
+    return (
+        <Text as='div' size='xxxs' line_height='s' className='range-text-container'>
+            <Localize
+                i18n_default_text='Range: {{min}} - {{max}} {{duration_unit}} '
+                values={{
+                    min,
+                    max: addComma(max),
```

Comment:
Don't we need comma for min values as well? What would happen if min value is very big or max value is very small?

Reply:
@akmal-deriv @maryia-deriv I'm not sure, because it's a `minimum duration`. I don't think it can be less then 1 as probably there will be no restriction `Range: 0.1 - 144 minutes `
I still can add comma 'just in case', what do you think?
<img width="520" alt="Screenshot 2023-09-26 at 11 56 00 AM" src="https://github.com/binary-com/deriv-app/assets/121025168/77c63a1b-dc01-4705-b8ba-3240a3ab9b96">

Reply:
Will add